packages/components/src/drop-zone/style.scss

```diff
@@ -1,3 +1,73 @@
+@keyframes fadeIn {
```

Comment:
I just recalled that these might conflict with other keyframes as this CSS is bundled into the final build. I'm assuming I should wait for and incorporate https://github.com/WordPress/gutenberg/pull/61940 after it's merged.

Reply:
I guess you mean you're waiting for #61940? If that's the case, should we mark the PR as a draft while waiting?

Reply:
Yeah, I realized after I set it as ready for review. However, AFAIK, the changes should be simple (only adding `:local`), so I think it's fine to leave it as ready for review, as it shouldn't affect the animation outcome. I can then locally scope the keyframes and do another round of tests before merging once #61940 is available on `trunk`.

Reply:
I namespace the identifiers manually for now: 540eafcda50be0a81cc0685a5321589bd6fadbef.